app/Http/Controllers/Marketing/Docs/Api/PruneAccountController.php

```
@@ -0,0 +1,20 @@
+declare(strict_types=1);
+
+namespace App\Http\Controllers\Marketing\Docs\Api;
+
+use App\Http\Controllers\Controller;
+use App\Jobs\RecordMarketingPageVisit;
+use Illuminate\Http\Request;
+use Illuminate\View\View;
+
+final class PruneAccountController extends Controller
+{
+    public function index(Request $request): View
```

PHPMD (2.15.0)

Avoid unused parameters such as '$request'. (undefined)

(UnusedFormalParameter)